themes/default/content/blog/iac-recommended-practices-structuring-pulumi-projects/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Iac Recommended Practices: Structuring Pulumi Projects"
+date: 2023-03-16T16:50:38-07:00
+meta_desc: The third post in the Zephyr series examines recommended practices around structuring Pulumi projects.
+meta_image: meta.png
+authors:
+    - scott-lowe
+    - christian-nunciato
+    - aaron-kao
+tags:
+    - best-practices
+    - cloud-engineering
+    - zephyr
+---
+
+This is the third post in a series of blog posts focused on Zephyr Archaeotech Emporium---our fictional company---and their use of Pulumi to manage their online retail store. In the first post, you saw how Zephyr initially decided to go with a single Pulumi project for managing deployments of their online retail store application. In this post, you'll see how Zephyr's use of Pulumi evolves as their company grows and evolves.<!--more-->
+
+The [first blog post](/blog/iac-recommended-practices-code-organization-and-stacks/) in the Zephyr series specifically mentions that the series will show "how Zephyr's use of Pulumi evolves as the company grows and their online retail store application changes to accommodate their growth." While the ultimate goal of this series is to discuss recommended practices for using Pulumi to manage a fairly complex containerized application, it's important to note that these recommended practices can and will change over time as the organizations using Pulumi also change and evolve.
```

Comment:
> it's important to note that these recommended practices can and will change over time as the organizations using Pulumi also change and evolve.

what does this mean? who does "organizations" refer to here? the hypothetical zephyr or you just saying that these won't always be the gold standard and these practices could be wrong if you read the post like 12 months from now?

Reply:
Looks like this could use some rewording. The intent behind this statement was to say that what might be "best" for any given customer at one point in time may not be "best" for that same customer 12 months later _if_ their circumstances have changed (where, like Zephyr, perhaps they've grown, perhaps they are using Pulumi for multiple initiatives, etc.). If you have suggestions on how to better phrase this, then please let me know!

Reply:
I've reworded this, see the version in e569521b and let me know what you think